src/statsmodels.jl

```diff
@@ -0,0 +1,549 @@
+
+
+### Util functions
+
+StatsBase.responsename(m) = coefnames(formula(m).lhs)
```

Comment:
I'd prefer we work with StatsAPI directly:
```suggestion
StatsAPI.responsename(m) = StatsAPI.coefnames(StatsAPI.formula(m).lhs)
```